### ISSUE TITLE
Build options propagate wrongly

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri#6699]
### Observed behavior

Example: The build switch `-s` isn't propagated to `kolibri-tools` but only appended to the final command.

```
$ yarn run build -s
yarn run v1.12.3
$ kolibri-tools build prod --file ./build_tools/build_plugins.txt && yarn run hashi-build -s
```

Work-around:

```
yarn run kolibri-tools build prod --file ./build_tools/build_plugins.txt -s
```

Might be a regression from #6268 

### Expected behavior

Ability to run single-core compiles.

### User-facing consequences

None

### Errors and logs

Nay

### Steps to reproduce

Run command

### Context

0.13.2.dev

**Comment by rtibbles** [comment]
Easiest thing to do would be to change the order of commands in the build command.

**Comment by rtibbles** [comment]
Fixed in #8322
